content/20.treatments.md

```diff
@@ -29,6 +29,10 @@ Antivirals have been used to treat XYZ diseases through [what mechanisms or appr
 #### Antivirals and Coronoviruses
 [Have antivirals been used to treat other coronaviruses? What went well/badly? How do these interventions work?]
 
+Neutralizing Antibodies
+Monoclonal antibodies targeting viral antigens have shown therapeutic efficacy against viruses like Ebola and SARS CoV.
+The first human neutralizing antibody against SARS-CoV-2 targeting the trimeric spike (S) glycoproteins has been developed.
+
```

Comment:
You can cite in manubot by adding in `[@doi:10.1101/2020.03.11.987958]`
(I pulled the DOI you added in #37 )